content/en/blog/2020/wasmhub-istio/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Extended and Improved WebAssemblyHub to Bring the Power of WebAssembly to Envoy and Istio
+subtitle: A hub to build, publish, share and deploy WebAssembly Envoy extensions.
+description: Community partner tooling of Wasm for Istio by Solo.ioo
```

Comment:
@bettyjunod you need to update the description field to end with a period to pass the tests. Should read something like:

`Community partner tooling of Wasm for Istio by Solo.io.`